free-2.md

```diff
@@ -5,40 +5,61 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
-
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+It can be difficult for script-writers (of TV, film, theater) to get a sense of how their dialogues sound before an initial read-through, especially when the dialogues are chaotic and involve many characters speaking lines at precise timings.  By having a simulation-type DSL that could perform read-throughs of dialogues, script-writers would have an easier time planning out chaotic dialogues before hiring voice actors.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Script-writers already write out their scripts via some sort of text editor.  With this DSL, they'd be able to assign specific timings for lines of dialogue.  Script-writers would then be able to export their script to an audio file and listen to a rough read-throughs of their dialogues without having to hire a crew of script-readers.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+To be honest I don't remember how I came up with this idea.  I just found a note on my iPhone from a week ago vaguely describing it.  What excites me is that it seems like something that could actually be of use to people.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Simulating dialogues that depend onTiming.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would interact with the language creating "scenes" of chaotic dialogue that are ideally not too long.  For each scene, the user would start by defining a list of actors.  The user would then type out the dialogue along with markings to indicate the timings of each line.
+
+Character Kevin = K  
+Character Maria = Ma  
+Character Marcus = Ms
+
+* K: Hey, wanna hear a jo- {-1}
+* Ms: Shut up! {10}
+* K: Uhh, you okay?
+* Ma: He's having a bad day.  Just leave him alone.  {-5} 
+* Ms: I said stop talking!  {20}
+* Ma: Oh.  {10}
+* K {-1} + Ma: Sorry.  
+
+Explanation: the number in braces at the end of each line represents the amount of silence before the next line (unit in this case is 1/10 of a second).  A negative value would indicate that the line is interrupted by the next line.  No value (see line 3) means that the next line will start after some default amount of time.  The last line will be spoken by both Kevin and Maria in near unison (Kevin will start one tenth of a second earlier than Maria) one full second (10 tenths of a second) after the previous line.
 
```

Comment:
This is a really cool idea. It seems like it could also be used by actors to mess with different timings of how to say things.